libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -980,6 +987,28 @@ bool RC_Channel::read_3pos_switch(RC_Channel::aux_switch_pos_t &ret) const
     return true;
 }
 
+// return switch position value as LOW, MIDDLE, HIGH
+RC_Channel::aux_switch_pos_t RC_Channel::get_aux_switch_pos() const
+{
+    RC_Channel::aux_switch_pos_t position = RC_Channel::LOW;
+    if (!read_3pos_switch(position)) {
```

Comment:
Ignore the result using `UNUSED_RESULT` and don't bother with the `if`.  It's obvious enough what's happening.

Reply:
fixed it.